.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -1,17 +1,19 @@
 ---
 name: Feature request
-about: Suggest an idea for this project
+about: Propose a new idea or start discussion about a new feature
 
 ---
 
 **Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+A clear and concise description of what the related problem is.
+E.g. "I'm always frustrated when [...]"
 
-**Describe the solution you'd like**
+**Describe the solution you would like**
 A clear and concise description of what you want to happen.
 
-**Describe alternatives you've considered**
+**Describe alternatives you have considered**
 A clear and concise description of any alternative solutions or features you've considered.
 
-**Additional context**
+**Additional context:**

```

Comment:
Why the `:` is necessary here? We do not use it in the other list items.

Reply:
You marked my comment as resolved but you didn't change anything here.

Reply:
> You marked my comment as resolved but you didn't change anything here.

Right... sorry. It seems I ended up not adding the change and force-pushing the same commit. It should be good now.